packages/build/src/signable-compiler.ts

```diff
@@ -22,6 +25,12 @@ class SignableCompiler {
    * @param {Function} exec - The boxednode compile function.
    */
   async compile(exec: Function) {
+    const cliReplRequire = Module.createRequire(path.resolve(__dirname, '..', '..', 'cli-repl', 'src'));
```

Comment:
Would it be possible to put `Module.createRequire` + `path.dirname` `pkgUp` in some function like `findModulePath`? Just so it is immediate what it does